src/pages/Main/CanvasDiv.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { canvasElement } from "./Canvas";
+import './CanvasDiv.css';
+
+interface CanvasDivProps {
+    
+}
+
+const CanvasDiv: React.FC<CanvasDivProps> = () => {
```

Comment:
Can we call this component canvasWrapper or canvasContainer?

Reply:
I chose canvasContainer. done